src/app/containers/SocialEmbed/index.jsx

```diff
@@ -47,7 +46,7 @@ const SocialEmbedContainer = ({ blocks, source }) => {
 
   const skipLink = {
     ...skipLinkTranslations,
-    endTextId: `skip-%provider%-content-${index}`,
+    endTextId: `skip-%provider%-content-${oEmbedPosition}`,
```

Comment:
Not sure if this is just an example? Anchor link text should ideally be named as per the destination e.g. "end-of-twitter-content-1"

Reply:
That's just the id on the element. The text is still correct. Does this look ok to you?

<img width="776" alt="Screenshot 2021-07-13 at 09 36 39" src="https://user-images.githubusercontent.com/4798332/125419845-c663d31d-488d-4edf-8cb1-264aa9f62a5b.png">

Reply:
The id forms part of the url, it should ideally be named as per the destination